test-engineering/contract-tests/client/tests/conftest.py

```diff
@@ -24,24 +28,32 @@
 )
 
 
+@pytest.fixture(scope="session", name="kinto_environment")
+def fixture_kinto_environment(request: Any) -> KintoEnvironment:
```

Comment:
@Trinaa do you know where this `request` fixture is defined? Having an `Any` type here isn't quite useful I wonder if we can find its actual type.

Reply:
@ncloudioj 

In my experience with this test suite, the request has been of type SubRequest from _pytest.fixtures (see https://github.com/mozilla-services/merino/pull/409/commits/6eee51e0715ddd8cee9bf9c35868cafa1db16552), but I've gathered this only from debug inspection, not from pytest documentation.

After speaking with @hackebrot I have opted to put `Any`, because, although unfortunate, I think it's accurate in this case, I think consistency in type hints is important and because importing from an internal module in pytest could make the solution unnecessarily vulnerable to breakage on a pytest update. 

thoughts?

Reply:
Sounds good to me, thanks for the context!